docs/running-on-yarn.md

```diff
@@ -495,6 +495,15 @@ To use a custom metrics.properties for the application master and executors, upd
   name matches both the include and the exclude pattern, this file will be excluded eventually.
   </td>
 </tr>
+<tr>
+  <td><code>spark.hadoop.callerContext</code></td>
```

Comment:
Yes. I have changed the config to spark.log.callerContext and moved the documentation to `spark/docs/configuration.md`.